_partials/newsgrid.html

```diff
@@ -11,18 +11,17 @@ <h1>24 maart eetdag</h1>
         </a>
     </div>
     
-   <div class="newsgrid-item large active has-content">
+   <div class="newsgrid-item large active has-content *large*">
          <a href="/news/2019-02-01-nieuwe-website/">
             <img src="/news/img/nieuwe-website.jpg" alt="Nieuwe Website!">
             <div class="newsgrid-item-overlay">
                 <h1>Nieuwe Website</h1>
                 <p>Zoals U merkt steekt onze website vanaf vandaag in een nieuw kleedje en is ook beter op smartphones te bekijken.</p>
-                <p><span class="btn btn-lg btn-primary">Meer weten</span></p>
             </div>
         </a>
     </div>   
 
-    <div class="newsgrid-item active has-content">
+    <div class="newsgrid-item active has-content *active*">
```

Comment:
> *active*

Wat wou je hier bereiken? Dit is geen geldige css classname en zal geen effect hebben